_build/templates/default/sass/login.scss

```diff
@@ -217,6 +217,36 @@ input[type="checkbox"] {
   }
 }
 
+#modx-login-btn {
+  border: 1px solid transparent;
+  &:hover {
+    background-color: $white;
+    color: $primary2;
+    box-shadow: none;
+    border: 1px solid $primary2;
+    transition: color .2s ease-out,
+    background-color .2s ease-out,
+    border .2s ease-out;
+  }
+  &:focus{
```

Comment:
Mark means that styles for the focused element should be the same as for the hover state. It needed for people who use a keyboard to navigate across the page instead of a mouse. So it should be:
```
&:hover, &:focus {
    ... styles
}
```